docs/howto/check-health.rst

```diff
@@ -33,15 +33,17 @@ To check the health status of your Airflow instance, you can simply access the e
     }
   }
 
-* The ``status`` of each component can be either "healthy" or "unhealthy".
-
-    * The status of ``metadatabase`` is depending on whether a valid connection can be initiated
-      with the database backend of Airflow.
-    * The status of ``scheduler`` is depending on when the latest scheduler heartbeat happened. If the latest
-      scheduler heartbeat happened 30 seconds (default value) earlier than the current time, scheduler component is
-      considered unhealthy. You can also specify this threshold value by changing
-      ``scheduler_health_check_threshold`` in ``scheduler`` section of the ``airflow.cfg`` file.
-
-* The response code of ``"/health"`` endpoint is not used to label the health status of the
-  application (it would always be 200). Hence please be reminded not to use the response code here
-  for health-check purpose.
+* The ``status`` of each component can be either "healthy" or "unhealthy"
+
+  * The status of ``metadatabase`` depends on whether a valid connection can be initiated with the database
+
+  * The status of ``scheduler`` depends on when the latest scheduler heartbeat was received
+
+    * If the last heartbeat was received more than 30 seconds (default value) earlier than the current time, the scheduler is
+      considered unhealthy
+    * This threshold value can be specified using the option ``scheduler_health_check_threshold`` within the
+      ``scheduler`` section in ``airflow.cfg``
+
+Please keep in mind that the HTTP response code of ``"/health"`` endpoint **should not** be used to determine the health
```

Comment:
```suggestion
Please keep in mind that the HTTP response code of ``/health`` endpoint **should not** be used to determine the health
```

Reply:
The same file has twice this issues.

Reply:
Still not all apostrophes are deleted. Look at the beginning of the file.:-)

Reply:
I am prepared patch for you
```bash
curl https://pastebin.com/raw/7ijpbhKi | git am
```

Reply:
my bad, missed the first set of quotes. :)